pages/Documents/MessagingChannels/ConnectToMessaging/api.md

```diff
@@ -0,0 +1,254 @@
+---
+pagename: API
+redirect_from:
+  - ConnectToMessaging.html
+Keywords:
+sitesection: Documents
+categoryname: "Messaging Channels"
+documentname: Connect To Messaging
+permalink: connect-to-messaging-api.html
+indicator: messaging
+---
+
+### Introduction
+
+Connect To Messaging (C2M) is a product offering from LivePerson allowing brands to offer consumers an option to deflect to messaging when they call into their IVR. C2M API serves as an intermediary between the brand’s IVR System and LivePerson Conversational Cloud, ensuring that the consumer is invited to join a conversation with an agent via eligible messaging channels. Once a consumer responds to a message from that channel, C2M ensures that the conversation is routed to an agent of the appropriate skill specified by the brands.
+
+
+### Getting Started
+
+1. Onboarding to C2M is mandatory before it.  
+2. Brand’s system should integrate with two C2M API endpoints, which are <strong><i>Eligibility</i></strong> and <strong><i>Invite</i></strong>. 
+  * <strong><i>Eligibility:</i></strong> Brands call this endpoint to check whether a consumer is reachable via a messaging channel.
+  * <strong><i>Invite:</i></strong> Brands call this endpoint to send a messaging invitation to transfer the customer from IVR to one of their supported channels.
+
+
+### API Specifications
+## API Domain
+
+C2M is deployed in three regions. **North America**, **EMEA**(Europe, Middle East and Africa), **APAC**(Asia Pacific). Use the domain api to identify the zone of C2M api which is to be used for an account.
+
+### Eligibility API
+
+Click [**Eligibility**](https://app.swaggerhub.com/apis-docs/nsavla/Connect_To_Messaging/1.0.0#/default/post_account__accountId__eligibility) to go through API spec and use example here to get started.
```

Comment:
I do not think we should use a like like https://app.swaggerhub.com/apis-docs/**nsavla**/Connect_To_Messaging in the developer site.